contracts/validator-manager/MessageSpec.md

```diff
@@ -1,45 +1,115 @@
 ## Warp Message Format Reference
 
+### `SubnetConversionMessage`
+
+- Description: Confirms conversion to a Permissionless Subnet on the P-Chain
+
+- Signed by: P-Chain
+
+- Consumed by: Validator Manager contract
+
+Specification:
+
+```
+SubnetConversionMessage
```

Comment:
Doesn't matter at all for this PR, but worth noting that markdown supports tables that provide auto-formatting in the future. It makes it easy to modify in the future without having to manually correct all of the spacing.